Lib/ssl.py

```diff
@@ -876,6 +876,20 @@ def getpeercert(self, binary_form=False):
         """
         return self._sslobj.getpeercert(binary_form)
 
+    def get_verified_chain(self):
+        """Returns verified certificate chain provided by the other
+        end of the SSL channel as a list of ``_ssl.Certificate``.
+        Return ``None`` if no certificates were provided.
```

Comment:
Since this returns a list, returning an empty [] list would be a simpler API.

Reply:
OK, I will fix that.